docs/config/@config.md

```diff
@@ -186,32 +186,27 @@ This will force the world to be restarted if the config is changed in the mod op
 
 Sub Categories
 --------------
-To make a Sub Category make an object, then add it as a static field in the parent categories class. The object's non-static fields will become configs in that Sub Category.
+To create a Sub Category make an object, then add it as a static field in the parent category's class. The object's member fields will become configs in that Sub Category.
 
 An example of how to setup a Sub Category:
 ```java
 @Config(modid = "modid")
 public class Configs {
-  public static SubCategory subcat = new SubCategory(59);
+  public static SubCategory subcat = new SubCategory(false);
 
   private static class SubCategory {
-    @Config.RangeInt(min = 0, max = 1000)
-    @Config.Comment("This is the FE/t used by the thing")
-    public int feForTheThing; 
+    public boolean someBool; 
 
-    public SubCategory(int defaultFE) {
-      feForTheThing = defaultFE;
+    public SubCategory(int someBool) {
```

Comment:
This constructor is still unnecessary fluff. You can define a default value by setting the field's value. bools also default to false, so it isn't even required here.

Reply:
ok :)

Reply:
Also I just realized you left `int` in that example constructor lol

Reply:
yup, i just woke up so i'm not completely with it... but idk if i ever am....